File: Documentation/5.2/Samples/csharp/Raven.Documentation.Samples/ClientApi/Operations/Server/GetClientConfig.cs
using System.Threading.Tasks;
using Raven.Client.Documents;
using Raven.Client.Documents.Operations.Configuration;
using Raven.Client.ServerWide.Operations.Configuration;

namespace Raven.Documentation.Samples.ClientApi.Operations.Server
{
    public class GetClientConfig
    {
        public GetClientConfig()
        {
            using (var store = new DocumentStore())
            {
                {
                    #region get_config
                    // Define the get client-configuration operation 
                    var getServerWideClientConfigOp = new GetServerWideClientConfigurationOperation();
                    
                    // Execute the operation by passing it to Maintenance.Server.Send
                    ClientConfiguration config = store.Maintenance.Server.Send(getServerWideClientConfigOp);
                    #endregion
                }
            }
        }
        
        public async Task GetClientConfigAsync()
        {
            using (var store = new DocumentStore())
            {
                {
                    #region get_config_async
                    // Define the get client-configuration operation 
                    var getServerWideClientConfigOp = new GetServerWideClientConfigurationOperation();
                    
                    // Execute the operation by passing it to Maintenance.Server.SendAsync
                    ClientConfiguration config = 
                        await store.Maintenance.Server.SendAsync(getServerWideClientConfigOp);
                    #endregion
                }
            }
        }
        
        private interface IFoo
        {
            /*
            #region syntax
            public GetServerWideClientConfigurationOperation()
            #endregion
            */
        }
    }
}

File: Documentation/5.4/Samples/nodejs/client-api/operations/Maintenance/Configuration/getClientConfig.js
import { DocumentStore } from "ravendb";

const documentStore = new DocumentStore();

async function getClientConfig() {
    {
        //region get_config
        // Define the get client-configuration operation 
        const getClientConfigOp = new GetClientConfigurationOperation();

        // Execute the operation by passing it to maintenance.send
        const result = await store.maintenance.send(getClientConfigOp);
        
        const configuration = result.configuration;
        //endregion
    }
}

{
    //region syntax_1
    const getClientConfigOp = new GetClientConfigurationOperation();
    //endregion

    //region syntax_2
    // Object returned from store.maintenance.send(getClientConfigOp):
    {
       etag,
       configuration // The configution object
    }

    // The configuration object:
    {
        identityPartsSeparator,
        etag,
        disabled,
        maxNumberOfRequestsPerSession,
        readBalanceBehavior,
        loadBalanceBehavior,
        loadBalancerContextSeed
    }
    //endregion
}
